src/frontend/citizen-portal/src/app/services/violation-ticket.service.ts

```diff
@@ -398,4 +409,67 @@ export class ViolationTicketService {
+    try {
+      let fields = ocrViolationTicket?.fields;
+      if (fields && fields !== undefined) {
+
+        if (this.getOcrViolationErrors(fields.violationTicketTitle)) { return true; }
+        if (this.getOcrViolationErrors(fields.ticket_number)) { return true; }
+        if (this.getOcrViolationErrors(fields.disputant_surname)) { return true; }
+        if (this.getOcrViolationErrors(fields.disputant_given_names)) { return true; }
+        if (this.getOcrViolationErrors(fields.drivers_licence_province)) { return true; }
+        if (this.getOcrViolationErrors(fields.drivers_licence_number)) { return true; }
+        if (this.getOcrViolationErrors(fields.violation_time)) { return true; }
+        if (this.getOcrViolationErrors(fields.violation_date)) { return true; }
+
+        // seems like a goofy way to process these but this is how the JSON parse returns it
+        // count 1
+        if (this.getOcrViolationErrors(fields["counts.count_no_1.description"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_1.act_or_regulation_name_code"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_1.is_act"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_1.is_regulation"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_1.section"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_1.ticketed_amount"])) { return true; }
+
+        // count 2
+        if (this.getOcrViolationErrors(fields["counts.count_no_2.description"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_2.act_or_regulation_name_code"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_2.is_act"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_2.is_regulation"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_2.section"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_2.ticketed_amount"])) { return true; }
+
+        // count 3
+        if (this.getOcrViolationErrors(fields["counts.count_no_3.description"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_3.act_or_regulation_name_code"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_3.is_act"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_3.is_regulation"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_3.section"])) { return true; }
+        if (this.getOcrViolationErrors(fields["counts.count_no_3.ticketed_amount"])) { return true; }
+
+        if (this.getOcrViolationErrors(fields.court_location)) { return true; }
+        if (this.getOcrViolationErrors(fields.detachment_location)) { return true; }
```

Comment:
you can create an array to include all this fields then check it in fromOCR function